plugins/@grouparoo/calculated-property/src/lib/profileProperty.ts

```diff
@@ -0,0 +1,68 @@
+import {
+  GrouparooCLI,
+  plugin,
+  Profile,
+  ProfilePropertyPluginMethod,
+} from "@grouparoo/core";
+import { NodeVM } from "vm2";
+
+async function calculateProfilePropertyValue(
+  customFunction,
+  profile: Profile,
+  propertyName: string
+): Promise<string> {
+  let populatedFunction;
+  try {
+    populatedFunction = await plugin.replaceTemplateProfileVariables(
+      customFunction,
+      profile,
+      false
+    );
+    //fail at every level if someone tries to require a library... this should never be allowed to hit vm.run
```

Comment:
processing the property in a run and allowing `vm.run` would fail if someone tried to require a library... since that would be an invalid config, we should let a user know before that point that their code is invalid.